Dev/VideoChatClient/Input/IInput.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+typedef std::vector<uint8_t> Buffer;
+
+class IInput
+{
+public:
+	IInput() = default;
+	virtual ~IInput() = default;
```

Comment:
Add deletion of the copy C'tor and operator=

Reply:
Why?